engine/api/event/kafka.go

```diff
@@ -0,0 +1,90 @@
+package event
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/Shopify/sarama"
+	"github.com/spf13/viper"
+
+	"github.com/ovh/cds/engine/log"
+	"github.com/ovh/cds/sdk"
+)
+
+var producer sarama.SyncProducer
+
+// Close close event system
+func Close() {
+	closeKafka()
+}
+
+func kafkaRoutine() bool {
+	if viper.GetString("event_kafka_broker_addresses") == "" ||
```

Comment:
I prefer not using viper strings in subpackage. we should handle them is main package